.github/workflows/security-scan.yml

```diff
@@ -1,14 +1,17 @@
 name: Security Scan
 
 on:
+  push:
+    branches:
+      - upgrade/go-version-MAY-2024
```

Comment:
You want to cleanup this?

Reply:
Oops, forgot to push that commit. Done, thanks